ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -679,28 +679,41 @@ initiator miniProtocolParameters version Apps {..} =
         })
       version
 
--- | A projection from 'NetworkApplication' to a server-side
--- 'OuroborosApplication' for the node-to-node protocols.
+-- | A bi-directional network applicaiton.
 --
--- See 'initiatorNetworkApplication' for rationale for the @_version@ arg.
-responder
+-- Implementation note: network currently doesn't enable protocols conditional
+-- on the protocol version, but it eventually may; this is why @_version@ is
```

Comment:
The new comment says,

> ...; this is why @_version@ is currently unused.

but this function does pass around its `version` parameter.

Maybe the comment was intended to describe the callbacks that are being passed `version` here.

Please clarify.